.editorconfig

```diff
@@ -1,5 +1,5 @@
 [*]
-indent_style = tabs
+indent_style = space
```

Comment:
😆  forgive my lack of clarity, @demariadaniel! when I said "leave ti alone" I meant "we shouldn't delete it".
Tabs is definitely a good suggestion, though
```suggestion
indent_style = tabs
```

Reply:
Added back in